src/components/feedback/Dialog/styles.less

```diff
@@ -0,0 +1,34 @@
+@import '~styles/mixins.less';
+
+.container {
+    padding: .grid-unit(2px)[] 0 .grid-unit(1px)[] 0;
+    border: 1px solid var(--color-black-alt4);
```

Comment:
This should probably be a color with an alpha? E.g. rgba(0, 0, 0, .2); or something. But since the dialogs should be controlled by the notification center, it might be better if the "scrim" is rendered by the notification center and not the dialog? Not sure about that one 🤔

Reply:
I'm not sure if I understand, this is not scrim styling

Reply:
My mistake. Misread border as background 🙈